build/tasks/mkrpm-task.coffee

```diff
@@ -30,9 +30,9 @@ module.exports = (grunt) ->
     rm rpmDir
     mkdir rpmDir
 
-    installDir = grunt.config.get('atom.installDir')
+    installDir = '/usr'
```

Comment:
Is there a reason this has to be hard-coded instead of allowing it to be specified?